ac/ac-train/src/Guidelines.jsx

```diff
@@ -0,0 +1,158 @@
+
+    <Typography gutterBottom>
+      In this interface you choose origin/destination cities from a map. The
+      first chosen city is the one you from which you start the travel and the
+      second is the destination. Other options are to be chosen from the form
+      under the map.
+    </Typography>
+  </React.Fragment>
+);
+
+export const CommandGuidelines = () => (
+  <React.Fragment>
+    <Typography variant="headline" gutterBottom>
+      Command line interface:
+    </Typography>
+
+    <Typography gutterBottom>
+      In this interface you write a command in the following format:
+    </Typography>
+    <pre>
+      {
+        'from [city] to [city] { young|half-fare|standard} {bike} {C1|C2} {one-way|return} '
+      }
+    </pre>
+    <Typography gutterBottom>
+      where cities are obligatory. Optional arguments are{' '}
+      {'{ young | half-fare | standard }'} stands for fare, {'{ bike }'} for
+      ticket with a bike, {'{ one-way | return }'} for the type of travel and{' '}
+      {'{ C1|C2 }'} is the first or the second class. Example:
+    </Typography>
+    <pre>from Zurich to Lausanne young C1 one-way</pre>
+    <Typography gutterBottom>
+      will order a ticket 1st class one-way from Zurich to Lausanne with a
+      discount for young people and without bike.
+    </Typography>
+  </React.Fragment>
+);
+
+export const StartingGuidelines = () => (
+  <React.Fragment>
+    <Typography variant="display2" gutterBottom>
+      Train Activity
+    </Typography>
+    <Typography gutterBottom>
+      You are about to see four different kinds of user interfaces for ordering
+      tickets. They will be presented in a random order. Your task is to order a
+      ticket, requested on the bottom of each page. Please note that options
+      one-way, standard fare, 2nd class and no bike are default in each
+      interface.
+    </Typography>
+    <CommandGuidelines />
+    <GraphicGuidelines />
+    <FormGuidelines />
+    <DragAndDropGuidelines />
+    <div style={{ marginTop: '20px' }} />
+  </React.Fragment>
+);
+
+export const SwitchGuidelines = ({ activity }: { activity: string }) => {
+  switch (activity) {
+    case 'start':
+      return <StartingGuidelines />;
+    case 'command':
+      return <CommandGuidelines />;
+    case 'form':
+      return <FormGuidelines />;
+    case 'dragdrop':
+      return <DragAndDropGuidelines />;
+    case 'graphical':
+      return <GraphicGuidelines />;
+    default:
+      return null;
+  }
+};
+
+const SpecificGuidelineController = ({
+  activity,
+  start,
+  step,
+  classes
+}: {
+  activity: string,
+  start: Function,
+  step: number,
+  classes: Object
+}) => (
+  <Grid container justify="center">
+    <Grid container className={step > 0 ? classes.instance : ''}>
+      <Grid item sm={12}>
+        {activity !== 'start' && (
+          <Typography variant="display2" gutterBottom>
+            Interface {step}
+          </Typography>
+        )}
+      </Grid>
```

Comment:
We do not need to label the interface number that they are on, so this can probably be deleted.